api/python/quilt3/admin/_graphql_client/__init__.py

```diff
@@ -0,0 +1,128 @@
+# Generated by ariadne-codegen
+
+from .add_roles import (
+    AddRoles,
+    AddRolesAdmin,
+    AddRolesAdminUser,
+    AddRolesAdminUserMutate,
+    AddRolesAdminUserMutateAddRolesInvalidInput,
+    AddRolesAdminUserMutateAddRolesOperationError,
+    AddRolesAdminUserMutateAddRolesUser,
+)
+from .base_client import BaseClient
+from .base_model import BaseModel, Upload
+from .client import Client
+from .create_user import (
+    CreateUser,
+    CreateUserAdmin,
+    CreateUserAdminUser,
+    CreateUserAdminUserCreateInvalidInput,
+    CreateUserAdminUserCreateOperationError,
+    CreateUserAdminUserCreateUser,
+)
+from .delete_user import (
+    DeleteUser,
+    DeleteUserAdmin,
+    DeleteUserAdminUser,
+    DeleteUserAdminUserMutate,
+    DeleteUserAdminUserMutateDeleteInvalidInput,
+    DeleteUserAdminUserMutateDeleteOk,
+    DeleteUserAdminUserMutateDeleteOperationError,
+)
+from .fragments import (
+    InvalidInputSelection,
+    InvalidInputSelectionErrors,
+    ManagedRoleSelection,
+    OperationErrorSelection,
+    UnmanagedRoleSelection,
+    UserSelection,
+    UserSelectionExtraRolesManagedRole,
+    UserSelectionExtraRolesUnmanagedRole,
+    UserSelectionRoleManagedRole,
+    UserSelectionRoleUnmanagedRole,
+)
+from .get_roles import GetRoles, GetRolesRolesManagedRole, GetRolesRolesUnmanagedRole
+from .get_user import GetUser, GetUserAdmin, GetUserAdminUser, GetUserAdminUserGet
+from .get_users import GetUsers, GetUsersAdmin, GetUsersAdminUser, GetUsersAdminUserList
+from .input_types import UserInput
+from .remove_roles import (
+    RemoveRoles,
+    RemoveRolesAdmin,
+    RemoveRolesAdminUser,
+    RemoveRolesAdminUserMutate,
+    RemoveRolesAdminUserMutateRemoveRolesInvalidInput,
+    RemoveRolesAdminUserMutateRemoveRolesOperationError,
```

Comment:
maybe we can tell the codegen to separate parts with an underscore to make it a little bit more readable? e.g. `RemoveRoles_Admin_User_Mutate_RemoveRoles_OperationError`

Reply:
nope, we can't AFAICT